### ISSUE TITLE
Credit The Author!!

### ISSUE THREAD

**Issue by jepunband** [vincenzogianfelice/RoguePortal#4]
so you stole the work of jeretc on github and made it your own?!!!?! .. well done!!!

https://github.com/jeretc/captive-portal

**Comment by s0meguy1** [comment]
He also stole it from me, but I'm not at all mad, he made it way, way better, and he spelled "Rogue" correctly haha

https://github.com/s0meguy1/RougeWifi

**Comment by gianfelicevincenzo** [comment]
@s0meguy1 I'm sorry you think this. I HAVE NOT STOLEN ANYTHING FROM ANYONE !. One day I was wandering around the github bulletin board and I came across your project. _YOU GIVEN ME ONLY THE INSPIRATION_!. I haven't copied anything .. it's not my style ... I do it for passion, so I push to give the best of myself. What would I have gained by copying us? **Nothing**! Unfortunately the commands only those ... the language is that! I repeat ... **do not in the slightest think that I did such a thing, NEVER!**. I've never done this in my "real" life and I certainly don't intend to start doing it now.
And sorry if I answer you just now, can I do something for you?.

... And sorry for my English.

**Comment by gianfelicevincenzo** [comment]
@jepunband let me understand ... stolen what? do you know what the etymology of the word "stolen" means !? Evidently not. More than anything else "incorporated" ... and yes ... **maybe I made a very small mistake regarding not having cited the source** ... yes .. **FOR THIS, I ASK VENIA!**.
But assuming that I cloned the project as it is, without touching anything, I think all the data regarding the developer is still inside! So a "smart" person can very well go through the code and check it. But then, do you know how many projects incorporate other projects? and what should we do ... lynch everyone !?
But assuming that the project is changing (so I will remove these "phishing pages" as I do not find them suitable for the purpose of this script ... they were just for testing), study the etymologies of the words ... vi You are welcome! If I had stolen it, I certainly wasn't publishing it on github or standing here to answer you!

Thanks and sorry for my English!

**Comment by s0meguy1** [comment]
> 
> 
> @s0meguy1 I'm sorry you think this. I HAVE NOT STOLEN ANYTHING FROM ANYONE !. One day I was wandering around the github bulletin board and I came across your project. _YOU GIVEN ME ONLY THE INSPIRATION_!. I haven't copied anything .. it's not my style ... I do it for passion, so I push to give the best of myself. What would I have gained by copying us? **Nothing**! Unfortunately the commands only those ... the language is that! I repeat ... **do not in the slightest think that I did such a thing, NEVER!**. I've never done this in my "real" life and I certainly don't intend to start doing it now.
> And sorry if I answer you just now.
> 
> ... And sorry for my English.

I was/am not at all mad and do not care at all. I stole bits of code from all over the place for mine, that is just how our community works. I think you did a FAR better job than I did with this anyway. I also do not want credit for anything, my version was complete shit! 

Keep up the good work!

**Comment by s0meguy1** [comment]
@vincenzogianfelice see my comment above - my original comment was not meant to sound like I was mad - I even mention in my original repo to use yours instead of mine:
![image](https://user-images.githubusercontent.com/25542926/102717912-39216080-42b3-11eb-815a-8414a1ac3aef.png)

**Comment by gianfelicevincenzo** [comment]
Yes, I saw it, thanks. But please, don't think about that in the slightest. I would never allow myself to copy a project. But I'm inspired! Feel free, if you like, to take my scripts and modify them (I don't think I will modify them, for now). This is the spirit of the community !.

I closed issue

**Comment by s0meguy1** [comment]
@vincenzogianfelice I completely understand and sorry if I sounded like I was mad, I am not at all - I was trying to make a joke (didn't go over well haha). We all do the same in our community, we get inspired like you said! And no, I'm done with this project. I made it a long time ago for a specific client/test, shared it and didn't feel like doing more with it. 

My code is out there for people to share!!
